include/highfive/bits/H5FileDriver_misc.hpp

```diff
@@ -15,6 +15,7 @@ namespace HighFive {
 #ifdef H5_HAVE_PARALLEL
 inline MPIOFileDriver::MPIOFileDriver(MPI_Comm comm, MPI_Info info) {
     add(MPIOFileAccess(comm, info));
+    add(MPIOCollectiveMD());
```

Comment:
As an alternative to `MPIOFileDriver` (which I personally would like to see deprecated), one can use:
```
    FileDriver adam;
    adam.add(MPIOFileAccess(MPI_COMM_WORLD, MPI_INFO_NULL));
    adam.add(MPIOCollectiveMD());
    File file(filename.str(), File::ReadWrite | File::Create | File::Truncate, adam);
```

Reply:
that makes a lot more sense to me, especially now that i have a better feel for "the highfive way" after taking a crack at the collective data work.